anoncreds/src/data_types/anoncreds/rev_reg.rs

```diff
@@ -73,14 +84,58 @@ impl RevocationStatusList {
         self.revocation_list.get(idx).as_deref().copied()
     }
 
+    pub(crate) fn update(
+        &mut self,
+        registry: Option<ursa::cl::RevocationRegistry>,
+        issued: Option<BTreeSet<u32>>,
+        revoked: Option<BTreeSet<u32>>,
+        timestamp: Option<u64>,
+    ) -> Result<(), error::Error> {
+        if let Some(reg) = registry {
+            self.registry = Some(reg)
+        };
```

Comment:
Wouldn't `self.registry = registry` suffice here?

Reply:
This is so that if `registry.is_none`, we do not reset the `self.registry` value